### ISSUE TITLE
SEO object no longer returns under WPGraphQL version v1.1.0+

### ISSUE THREAD

**Issue by joshsilb** [moonmeister/wp-graphql-seopress#9]

**Comment by moonmeister** [comment]
Hey, I'm not using seopress currently and don't have a lot of spare time. If you want to contribute a PR I'm happy to review, merge, and release but am not able to dedicate the time to the actual debugging and fixing.

**Comment by DeathCamel58** [comment]
Hey idkidm,

I'm not 100% sure if I'm experiencing the same issue as you, but it seems pretty similar.

When I query:
```
query MyQuery {
  posts {
    edges {
      node {
        id
        seo {
          canonicalUrl
          metaDesc
          metaRobotsNofollow
          metaRobotsNoindex
          metaTitle
          opengraphDescription
          opengraphTitle
          twitterDescription
          twitterTitle
          twitterImage {
            uri
          }
          opengraphImage {
            uri
          }
        }
      }
    }
  }
}
```

I get the response:
```
{
  "errors": [
    {
      "debugMessage": "Argument 1 passed to WPGraphQL\\Data\\DataSource::resolve_post_object() must be of the type int, string given, called in /var/www/html/wp-content/plugins/wp-graphql-seopress/wp-graphql-seopress.php on line 79",
      "message": "Internal server error",
      "extensions": {
        "category": "internal"
      },
      "locations": [
        {
          "line": 6,
          "column": 9
        }
      ],
      "path": [
        "posts",
        "edges",
        0,
        "node",
        "seo"
      ],
      "trace": [
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql-seopress/wp-graphql-seopress.php",
          "line": 79,
          "call": "WPGraphQL\\Data\\DataSource::resolve_post_object()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/src/Utils/InstrumentSchema.php",
          "line": 159,
          "function": "{closure}()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/src/Utils/InstrumentSchema.php",
          "line": 159,
          "call": "WPGraphQL\\Utils\\InstrumentSchema::WPGraphQL\\Utils\\{closure}()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 623,
          "call": "WPGraphQL\\Utils\\InstrumentSchema::WPGraphQL\\Utils\\{closure}()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 550,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveFieldValueOrError()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1195,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1145,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1106,
          "call": "GraphQL\\Executor\\ReferenceExecutor::collectAndExecuteSubfields()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 793,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeObjectValue()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 654,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1195,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1145,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1106,
          "call": "GraphQL\\Executor\\ReferenceExecutor::collectAndExecuteSubfields()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 793,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeObjectValue()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 654,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 887,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 761,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeListValue()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 654,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1195,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1145,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1106,
          "call": "GraphQL\\Executor\\ReferenceExecutor::collectAndExecuteSubfields()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 793,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeObjectValue()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 651,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/Promise/Adapter/SyncPromise.php",
          "line": 148,
          "call": "GraphQL\\Executor\\ReferenceExecutor::GraphQL\\Executor\\{closure}()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/Promise/Adapter/SyncPromise.php",
          "line": 53,
          "call": "GraphQL\\Executor\\Promise\\Adapter\\SyncPromise::GraphQL\\Executor\\Promise\\Adapter\\{closure}()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Executor/Promise/Adapter/SyncPromiseAdapter.php",
          "line": 149,
          "call": "GraphQL\\Executor\\Promise\\Adapter\\SyncPromise::runQueue()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Server/Helper.php",
          "line": 214,
          "call": "GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter::wait()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/vendor/webonyx/graphql-php/src/Server/StandardServer.php",
          "line": 136,
          "call": "GraphQL\\Server\\Helper::executeOperation()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/src/Request.php",
          "line": 629,
          "call": "GraphQL\\Server\\StandardServer::executeRequest()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/src/Router.php",
          "line": 468,
          "call": "WPGraphQL\\Request::execute_http()"
        },
        {
          "file": "/var/www/html/wp-content/plugins/wp-graphql/src/Router.php",
          "line": 270,
          "call": "WPGraphQL\\Router::process_http_request()"
        },
        {
          "file": "/var/www/html/wp-includes/class-wp-hook.php",
          "line": 292,
          "call": "WPGraphQL\\Router::resolve_http_request()"
        },
        {
          "file": "/var/www/html/wp-includes/class-wp-hook.php",
          "line": 316,
          "call": "WP_Hook::apply_filters()"
        },
        {
          "file": "/var/www/html/wp-includes/plugin.php",
          "line": 551,
          "call": "WP_Hook::do_action()"
        },
        {
          "file": "/var/www/html/wp-includes/class-wp.php",
          "line": 388,
          "function": "do_action_ref_array()"
        },
        {
          "file": "/var/www/html/wp-includes/class-wp.php",
          "line": 750,
          "call": "WP::parse_request()"
        },
        {
          "file": "/var/www/html/wp-includes/functions.php",
          "line": 1291,
          "call": "WP::main()"
        },
        {
          "file": "/var/www/html/wp-blog-header.php",
          "line": 16,
          "function": "wp()"
        },
        {
          "file": "/var/www/html/index.php",
          "line": 17,
          "function": "require('/var/www/html/wp-blog-header.php')"
        }
      ]
    }
  ],
  "data": {
    "posts": {
      "edges": [
        {
          "node": {
            "id": "cG9zdDox",
            "seo": null
          }
        }
      ]
    }
  },
  "extensions": {
    "debug": []
  }
}
```

I've fixed that issue for myself locally, and am about to file a PR.

**Comment by DeathCamel58** [comment]
As a note, the same query now responds this in my dev enviornment:
```
{
  "data": {
    "posts": {
      "edges": [
        {
          "node": {
            "id": "cG9zdDox",
            "seo": {
              "canonicalUrl": "http://127.0.0.1/cononical-url",
              "metaDesc": "SEO Description!",
              "metaRobotsNofollow": "",
              "metaRobotsNoindex": "",
              "metaTitle": "SEO Title",
              "opengraphDescription": "Facebook Description",
              "opengraphTitle": "Facebook Title",
              "twitterDescription": "Twitter Description",
              "twitterTitle": "Twitter Title",
              "twitterImage": null,
              "opengraphImage": null
            }
          }
        }
      ]
    }
  },
  "extensions": {
    "debug": []
  }
}
``

**Comment by moonmeister** [comment]
released in v1.2
